tests/ui/test_opportunities_tab.py

```
@@ -0,0 +1,112 @@
+
+_SCRIPT = textwrap.dedent(
+    f"""
+    import sys
+    sys.path.insert(0, {repr(str(_PROJECT_ROOT))})
+    from ui.opportunities_tab import render_opportunities_tab
+    render_opportunities_tab()
+    """
+)
+
+
+def _collect_opportunities(queue: "Queue") -> None:
```

Ruff (0.13.1)

Undefined name `Queue`

(F821)